eclass/distutils-r1.eclass

```diff
@@ -1405,9 +1405,9 @@ distutils_pep517_install() {
 	fi
 	printf '%s\n' "${cmd[*]}"
 	local wheel=$(
-		"${cmd[@]}" 3>&1 >&2 || die "Wheel build failed"
+		"${cmd[@]}" 3>&1 >&2 || die "Build system error: build failed"
 	)
-	[[ -n ${wheel} ]] || die "No wheel name returned"
+	[[ -n ${wheel} ]] || die "Build system error: no wheel name returned"
```

Comment:
This is an inferior version of my patch, without my signoff. :(

Reply:
8d0f2de98faf0a7ac82dd9b1b465550722647163

In particular, the wheel name is an obscure way to describe it -- it's an output filename, and it's an issue because the process has nominally succeeded with a successful return code.

Reply:
I have been waiting for your reply to the comment on your patch for a month or so. I don't really care. "Internal error" makes little sense to me, it's build system misbehaving.

Reply:
Ok, reverted the change. I don't need your accusations.

Reply:
It's a provenance thing, it's not an accusation - you obviously did it because of the other patch, so mention it.

But really, don't leave comments on commits, nobody looks at them?

Reply:
So perhaps you should submit changes correctly instead of throwing links to detached commits on IRC.